site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
That might not be the case for BuckleScript. We might need discuss best practice for Buckle. So for people who use typescript, what do they distribute to the clients? if they only distribute typescript, then they may need add typescript as a dependency  in `postinstall`